app/scripts/modules/core/application/modal/pagerDutySelectField.directive.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.core.application.pagerDutySelectField.directive', [
+    require('../service/pagerDuty.read.service.js'),
+  ])
+  .directive('pagerDutySelectField', function() {
```

Comment:
Can you change this to a `component` and rename files/module names from `.directive` to `.component`?

Components are very similar, so you won't need to make too many changes:
 * `.directive('pagerDutySelectField`, function() {` becomes `.component('pagerDutySelectField', {`
 * can drop the `return {`
 * `controller` can be an anonymous function instead of a separate thing (I won't make you convert this to Typescript, although it would be nice)
 * no need for `scope` - it's implicit
 * `bindToController` becomes `bindings`
 * can drop `controllerAs` and replace `vm` with `$ctrl` in the template